packages/google_maps_flutter/google_maps_flutter_web/README.md

```diff
@@ -50,6 +50,19 @@ To request multiple libraries, separate them with commas:
 
 Now you should be able to use the Google Maps plugin normally.
 
+## Marker clustering
+
+If you need marker clustering support, modify the <head> tag to load the [js-markerclusterer](https://github.com/googlemaps/js-markerclusterer#install) library, like so:
+
```

Comment:
We should specify the exact version(s) of the library that the js-interop code of the plugin supports.